src/platform/windows/display.h

```diff
@@ -338,7 +357,53 @@ namespace platf::dxgi {
     texture2d_t old_surface_delayed_destruction;
     std::chrono::steady_clock::time_point old_surface_timestamp;
     std::variant<std::monostate, texture2d_t, std::shared_ptr<platf::img_t>> last_frame_variant;
+  };
 
-    std::atomic<uint32_t> next_image_id;
+  class uwp_capture_t {
```

Comment:
I'd prefer `wgc` rather than `uwp` for the name of the classes and options, since it's not really tied to UWP apps.